<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Calculator</class>
 <widget class="QMainWindow" name="Calculator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>417</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calculator</string>
  </property>
  <property name="locale">
   <locale language="English" country="Canada"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="font">
    <font/>
   </property>
   <widget class="QLineEdit" name="Display">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>441</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
     background-color: gray; border: 1px solid gray;
	color: #ffffff;}</string>
    </property>
    <property name="text">
     <string>0.0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>7</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_8">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>8</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_9">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>9</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Divide">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #E0FFFF; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #AFEEEE; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>/</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_MemAdd">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>210</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #E0FFFF; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #AFEEEE; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>M+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_5">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>5</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_6">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Multiply">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #E0FFFF; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #AFEEEE; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>*</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_MemClear">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #E0FFFF; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #AFEEEE; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>M-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_1">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Clear">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>C</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_0">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_3">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Add">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #E0FFFF; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #AFEEEE; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Subtract">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #E0FFFF; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #AFEEEE; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_MemGet">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>290</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #E0FFFF; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #AFEEEE; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>M</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Equals">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #E0FFFF; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #AFEEEE; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="locale">
     <locale language="English" country="Canada"/>
    </property>
    <property name="text">
     <string>=</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Pow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>x2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_DecimalPoint">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>330</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font/>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton {
     background-color: #DCDCDC; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}

QPushButton:pressed {
    background-color: #A9A9A9; border: 1px solid gray;
	border-radius: 0px;
	padding: 5px;
}</string>
    </property>
    <property name="text">
     <string>.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="History">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>441</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>26</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit {
     background-color: #E0FFFF; border: 1px solid gray;
	color: #D3D3D3;}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>488</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
